packages/frontend/src/pages/Tile/hooks/useUpdateRow.tsx

```diff
@@ -1,4 +1,4 @@
-import { useCallback, useState } from 'react'
+import { Dispatch, SetStateAction, useCallback, useState } from 'react'
```

Comment:
n00b qn: do we need to explicitly add `type` to help tree shaking?

```suggestion
import { type Dispatch, type SetStateAction, useCallback, useState } from 'react'
```